library/src/main/java/com/eggheadgames/android_analytics_helper/Analytics.java

```diff
@@ -25,85 +29,109 @@
     public static final String PUZZLE_ID = "puzzle_id";
     public static final String HINT_COUNT = "hint_count";
     public static final String TOTAL_SECONDS = "total_seconds";
+    public static final String VOLUME_STATE = "volume_state";
+    public static final String LOCATION = "location";
+    public static final String MESSAGE = "message";
+    public static final String FATAL = "fatal";
+    public static final String THREAD_NAME = "thread_name";
+    public static final String DESCRIPTION = "description";
+    public static final String VOLUME = "volume";
+    public static final String SCREEN_NAME = "screen_name";
+
+    private MobileAnalyticsManager analytics;
```

Comment:
Don't think it should be `static`. As long as this field for internal use only we can set instance of `Analytics.java` to be static, so instance of `MobileAnalyticsManager` will live inside of it